

On ppc64, s64 is `long'.

In file included from fs/jfs/jfs_xtree.h:25,
                 from fs/jfs/jfs_incore.h:26,
                 from fs/jfs/super.c:29:
fs/jfs/jfs_btree.h: In function `BT_STACK_DUMP':
fs/jfs/jfs_btree.h:151: warning: long long unsigned int format, s64 arg (arg 2)

Cc: Dave Kleikamp <shaggy@austin.ibm.com>

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-power4-akpm/fs/jfs/jfs_btree.h |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN fs/jfs/jfs_btree.h~jfs-warning-fix fs/jfs/jfs_btree.h
--- 25-power4/fs/jfs/jfs_btree.h~jfs-warning-fix	2004-06-19 22:26:32.694183288 -0700
+++ 25-power4-akpm/fs/jfs/jfs_btree.h	2004-06-19 22:26:42.968621336 -0700
@@ -147,7 +147,7 @@ static inline void BT_STACK_DUMP(struct 
 	printk("btstack dump:\n");
 	for (i = 0; i < MAXTREEHEIGHT; i++)
 		printk(KERN_ERR "bn = %Lx, index = %d\n",
-		       btstack->stack[i].bn,
+		       (long long)btstack->stack[i].bn,
 		       btstack->stack[i].index);
 }
 
_
